dice:
  dice_top_desired_value: 4